repo-info.json

```diff
@@ -0,0 +1,7 @@
+{
+  "is_in_production": false,
```

Comment:
it was `true` for the UI

Reply:
Based on the conversation, it should be true (https://emarsys.slack.com/archives/C632Z86BW/p1627997350019800)